a/abyss/abyss_ubi_8.5.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package	: abyss
+# Version	: v2.3.5
+# Source repo	: https://github.com/bcgsc/abyss.git
+# Tested on	: UBI: 8.5
+# Language      : Go 
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer	: Muskaan Sheik <Muskaan.Sheik@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME=abyss
+PACKAGE_VERSION=${1:-2.3.5}
+PACKAGE_URL=https://github.com/bcgsc/abyss.git
+
+yum install -y sudo
+sudo yum update -y
```

Comment:
@MuskaanSheik  This script is supported for root or non-root user? why are we using sudo at some places only?

Reply:
@nikhil-kalbande It isn't needed. Removed that. I was following the steps in the abyss repository. But works the same without sudo so removed it